test/factories/review_reminder.rb

```diff
@@ -11,23 +11,20 @@
 
     trait(:reminder_due) do
       review_at { Time.zone.today }
-      reminder_sent_at { nil }
+      # Attempting to set review_at to today usually causes a validation error, because review date should be set to a future date
+      to_create { |instance| instance.save(validate: false) }
     end
 
     trait(:reminder_sent) do
-      review_at { Time.zone.today }
       reminder_sent_at { Time.zone.now }
     end
 
     trait(:due_but_never_published) do
-      review_at { Time.zone.today }
-      reminder_sent_at { nil }
       document { create(:document, editions: [build(:draft_edition, first_published_at: nil)]) }
     end
```

Comment:
It might be worth renaming this to reflect it no longer affects the 'due' date.

There's an argument to say this should be a trait on the `documents` factory rather than here, but I think it's only really relevant to this set of unit tests.

```suggestion
    trait(:document_never_published) do
      document { create(:document, editions: [build(:draft_edition, first_published_at: nil)]) }
    end
```

Reply:
Ah yes, good spot